source/mir/random/flex/internal/transformations.d

```diff
@@ -54,7 +62,8 @@ in
 }
 body
 {
-    import mir.internal.math: pow, exp, copysign;
+    import mir.internal.math: pow, copysign;
+    import std.math : exp;
```

Comment:
that's how I know about it. Only a tiny difference can yield a different splitting in the Tinflex algorithm :/